### PR TITLE
Use base template in eNikshay repeater report

### PR DESCRIPTION
**Pull request by proteusvacuum**
I'm still getting the offline generation error, even after https://github.com/dimagi/commcare-hq/pull/17203
I don't care about all the javascript or anything, and the report looks exactly the same with this template, and I suspect the other template is what is causing the error.

@benrudolph

### CODE REVIEW

File: custom/enikshay/reports/repeaters.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def _get_repeaters(self):
 class ENikshayForwarderReport(DomainForwardingRepeatRecords):
     name = 'eNikshay Forwarder Report'
     base_template = 'reports/base_template.html'
+    asynchronous = True
```
Comment (proteusvacuum) [inline custom/enikshay/reports/repeaters.py]: I think this is the golden ticket... maybe this is obvious, but do `emailable` reports also have to be marked `asynchronous`?

Comment (benrudolph) [inline custom/enikshay/reports/repeaters.py]: interesting... i don't know why that would be the case

**Review by benrudolph — APPROVED** [review]
curious how you're testing this

**Comment by proteusvacuum** [comment]
setting up a saved report locally, then seeing what gets generated when I send it manually (now with all these changes there is no javascript included in the report). Obviously I can't get the `OfflineGenerationError`, but am making an educated guess as to what is causing it.

**Comment by benrudolph** [comment]
👍 there are some instructions for settings up django to use compression in the readme, but what you're doing sounds good and most time effective

**Pull request merged**